src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/SelectColumnPanel.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.ui.stepsdesign;
+
+import io.cdap.cdap.ui.utils.Helper;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.SeleniumDriver;
+import io.cdap.e2e.utils.WaitHelper;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class SelectColumnPanel {
+  @Then("Click on the Structure icon")
+  public void clickOnTheStructureIcon() {
+    WaitHelper.waitForPageToLoad();
```

Comment:
This `WaitHelper.waitForPageToLoad();` is not necessary for every step. Only the first or any step where the page is actually being loaded. In order to avoid having one liner steps, try to combine steps instead. use something like `Change data type` as the step and have it as:
```java
@Then("Change data type")
public void changeDataType() {
  ElementHelper.clickOnElement(Helper.locateElementByTestId("toolbar-icon-structure"));
  ElementHelper.clickOnElement(Helper.locateElementByTestId("menu-item-changeDatatype"));
  ElementHelper.clickOnElement(Helper.locateElementByTestId("menu-item-string"));
}
```

Reply:
Please remove the unnecessary `WaitHelper.waitForPageToLoad();` in the other classes as well. I won't repeat this comment in those classes.

Reply:
Removed `WaitHelper.waitForPageToLoad();` in all classes where it is not needed.